lang/py/avro/test/test_schema.py

```diff
@@ -713,6 +713,22 @@ def test_parse_invalid_symbol(self):
         except avro.errors.InvalidName:  # pragma: no coverage
             self.fail("When enum symbol validation is disabled, an invalid symbol should not raise InvalidName.")
 
+    def test_parse_duplicate_symbol(self):
+        duplicate_symbol_schema = json.dumps({"type": "enum", "name": "AVRO3573", "symbols": ["A", "A", "B", "C", "D"]})
+        with self.assertRaisesRegex(
+            avro.errors.AvroException, r"Duplicate symbol: \[\'A\'\]", msg="When enum symbol has a duplicate, AvroException should raise."
+        ):
+            avro.schema.parse(duplicate_symbol_schema)
+
+    def test_parse_duplicate_symbols(self):
+        duplicate_symbols_schema = json.dumps({"type": "enum", "name": "AVRO3573", "symbols": ["A", "A", "B", "C", "C", "D"]})
+        with self.assertRaisesRegex(
+            avro.errors.AvroException,
+            r"Duplicate symbols: (\[\'A\', \'C\'\])|(\[\'C\', \'A\'\])",
```

Comment:
Is there a case where the resulting order in error message would be backwards?
If not I'd prefer for readability that error message and assertion would be exact. E.g.
```python
    with self.assertRaises(avro.errors.AvroException, msg="Should raise AvroException") as exc_info:
            avro.schema.parse(duplicate_symbols_schema)
    assert str(exc_info.exception) == "Duplicate symbols: (['A', 'C'])"
```